### DISCUSSION TITLE
"Loop Disabled" status on XDrip Display

### DISCUSSION THREAD

**Discussion by roco5** [NightscoutFoundation/xDrip#3116] [Bug reports]
Trying to figure out why this line " Loop Disabled" is on my xdrip. The AAPS loop is not disabled and is working fine. And it looks like xdrip is pulling other AAPS data over on the line below. So why is it showing "loop disabled"?

![385840286_10161207556933086_2258213398960252037_n](https://github.com/NightscoutFoundation/xDrip/assets/6946920/87ef1b0c-b421-4562-b3e5-8e9dc15cae00)

**Comment by roco5** [comment]
Maybe this is because I am using NSClient and this is not the phone with the full AAPS on it?

**Reply by jamorham** [reply]
Can you describe your setup in terms of phones and what is connected to what?

**Comment by Navid200** [comment]
@jamorham @tzachi-dar Looking at the code, I can only find "Loop Disabled" in test and under wear integration.
https://github.com/NightscoutFoundation/xDrip/blob/39370edfa35b5ce79fe5fba2c17dd1cffaed09db/app/src/test/java/com/eveningoutpost/dexdrip/wearintegration/ExternalStatusServiceTest.java#L24-L28

Is a user supposed to see this?

**Comment by roco5** [comment]
Sure:

Google Pixel 6a follower phone running current versions of NSCLIENT and
XDRIP.  XDRIP gets pump data from AAPS I presume, but we also have a
Nightscout server running on Heroku as well.

Kindest Regards,
Rob Conley
